base/src/main/java/com/smartdevicelink/protocol/enums/FunctionID.java

```diff
@@ -132,6 +132,9 @@
     ON_RC_STATUS(32785, "OnRCStatus"),
     ON_APP_SERVICE_DATA(32786, "OnAppServiceData"),
     ON_SYSTEM_CAPABILITY_UPDATED(32787, "OnSystemCapabilityUpdated"),
+    ON_SUBTLE_ALERT_PRESSED(32788, "OnSubtleAlertPressed"),

```

Comment:
Unit tests are needed for `OnSubtleAlertPressed`.

Reply:
@santhanamk this was displayed because of unresolved merge conflicts with dev branch. In the latest version after the update it is not marked as change, the FunctionID.java was aligned according to the latest specification